### DISCUSSION TITLE
I'm sorry for asking this, but is there a step by step Windows guide?

### DISCUSSION THREAD

**Discussion by poker620** [sct/overseerr#643] [Help] [answered]
Hi!

I've spent about an hour installing (or trying to) Docker on Windows 10 and longer trying to look at comments for help, but I'm pretty lost.  

I'm running Plex on external hard drives connected to a Windows 10 PC, super simple setup.   I've read the guide several times and all of the reddit r/plex comments on this but it's over my head.  One problem I see is that there seems to be lots of options for Docker, I don't care and will literally only use it to get this running on my Windows machine.  I'm not sure if this matters, but I wouldn't even need to have people request from outside my network, and/or don't care if the address is just my IP address.

So, ELi5 please, can someone provide step by step instructions that just use the easiest options for my setup?  For example where do I get Docker, what options do I pick, how do I run overseerr from the code given in the guide?  

Thank you SO much!

**Comment by shavedbroom** [comment]
install git for windows, when thats done install yarn by opening a git terminal and writing pip install yarn
Download the overseer source from here extract to something like C:\overseerr, in your git window or open a cmd window and head to overseerr directory "cd C:\overseerr" enter yarn install, then: yarn build, and then yarn start and it runs

**Reply by austinbarberrepo** [reply]
I tried run the yarn install, but I get this message " 'node' is not recognized as an internal or external command, operable program, or batch file"

**Reply by shavedbroom** [reply]
yes you need node, here: https://nodejs.org/en/download/

**Reply by austinbarberrepo** [reply]
Edit: I think I got it. I had to run this command "npm install -g win-node-env". Thanks for your help!

I get this when I run yarn start:

![image](https://user-images.githubusercontent.com/26965604/104646971-30883400-567f-11eb-8b3c-4e9bf7fac407.png)

**Reply by austinbarberrepo** [reply]
Can I use the yarn run command to have it start this app on reboot or is there a different way to do it? Sorry I’m new to use yarn.

**Reply by poijoy** [reply]
For docker install and setup, they provide a really good educational resource on their site, start here and take it slow:

https://docs.docker.com/get-started/

Once you've got docker installed and working, you can install overseer with this command:

docker run -d -e LOG_LEVEL=info -e TZ=Pacific/Auckland -p 5055:5055 -v C:\Docker\Overseerr\config:/app/config --restart unless-stopped sctx/overseerr

Change the TZ to your locale based in this list:

https://en.wikipedia.org/wiki/List_of_tz_database_time_zones

**Reply by austinbarberrepo** [reply]
Do I need to run "yarn build" everytime there is a new release to see the new features? The software shows that it's on the latest version. Should I just switch to the docker version to be more stable? Thank you.

**Reply by samwiseg0** [reply]
> Do I need to run "yarn build" everytime there is a new release to see the new features? The software shows that it's on the latest version. Should I just switch to the docker version to be more stable? Thank you.

yes. This method is unsupported and will break in some cases. I suggest taking the time to figure out how to install docker. We have instructions in the docs. 

https://docs.overseerr.dev

**Reply by pir8radio** [reply]
Overseer looks promising, it's a bit disappointing its shoved in a docker container.

**Comment by samwiseg0** [comment, marked as answer]
Unfortunately, we do not support building the app yourself. You have to use one of the supported install methods. If you want to build and use the app on your own we cannot offer any assistance. We do not offer any documentation for this reason.

**Comment by austinbarberrepo** [comment]
Are you familiar with running docker inside of a VM? Or is it possible to
run docker on my host machine and type it back to the VM's IP
address?Sorry, I have everyone on a VM so that I can use the VPN inside of
it while not affecting other apps on my host machine. If not, no worries.
Just thought I'd ask. Thank you.

**Reply by samwiseg0** [reply]
You can run docker in a VM. There are tons of guides out there for installing docker. I suggest you give them a try.
